eras/shelley/formal-spec/address-binary.tex

```diff
@@ -105,4 +105,4 @@ \subsection{Payloads}
 
 The variable-length encoding used in pointers addresses is the base-128 representation
 of the number, with the the most significant bit of each byte indicating continuation.
```

Comment:
```suggestion
of the number, with the most significant bit of each byte indicating continuation.
```
This is probably the best opportunity to fix this minor grammar error too.

As this is formally called [VQL](https://en.wikipedia.org/wiki/Variable-length_quantity), it might be worth adding a note about that too just to be extra clear?

Reply:
thanks @craphtex! I think making mention of the name VLQ sounds helpful.